src/iris/api.py

```diff
@@ -1383,6 +1412,16 @@ def on_post(self, req, resp, incident_id):
             raise HTTPBadRequest('"owner" field required')
 
         with db.guarded_session() as session:
+            owner_valid = \
```

Comment:
This is a little tricky because we allow users to "unclaim" by passing NULL as the "owner" key. Can you only do the check if the owner isn't NULL/None, and add the NULL functionality to the route comment doc?

Also, can you avoid the trailing backslash to terminate the line?